packages/experiments/src/components/Recommendation/Recommendation.styles.ts

```diff
@@ -0,0 +1,47 @@
+import { ICardComponentCustomizationStyles, IRecommendationStyles } from './Recommendation.types';
+import { IStyle, mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+
+const cardFrameColor = 'rgba(0, 120, 215, 1)';
+const recommendationBaseColor = 'rgba(0,0,0,1)';
+const commonFontFamily = 'Segoe UI';
+const commonFontWeight = 'bold';
+const largeFontSize = 28;
+const regularFontSize = 12;
+const containerSpaceMargin = 24;
```

Comment:
Does fabric define these in a common place anywhere? We should import that instead of defining these in each component.

Reply:
No fabric doesn't define them specifically , as the font size itself varies across common controls. These are defined here as they are constant across Recommendation control.

Reply:
No fabric doesn't define them specifically , as the font size itself varies across common controls. These are defined here as they are constant across Recommendation control.

---
In reply to: [196474548](https://github.com/OfficeDev/office-ui-fabric-react/pull/5252#discussion_r196474548) [](ancestors = 196474548)